website/tutorial/basic/12-summary.mdx

```diff
@@ -0,0 +1,11 @@
+---
+id: 11-summary
+title: Summary
+sidebar_label: Summary
+---
+
+In this tutorial, you learned how to build a GraphQL server from scratch using graphql-yoga.
+
+The resolvers of your GraphQL server are implemented using Prisma Client which is responsible for database access.
+
+Congratulations on completing the tutorial! We can't wait to see what you build.
```

Comment:
Maybe adding a "Call to action" here? With share ware you learned [here](twitter # I don't know what)

And reminding the repo of the demo code?

Reply:
love the idea!